aiida/orm/nodes/data/msonable.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+"""Data plugin for classes that implement the ``MSONable`` class of the ``monty`` library."""
+import importlib
+
+from monty.json import MSONable
```

Comment:
This isn’t used no?

Reply:
Oh no actually it is, but monty is not a direct dependency of aiida is it?

Reply:
This is no longer applicable: dependency has been removed.